client-react/src/settings/index.js

```diff
@@ -1,61 +1,84 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import { device } from "../common/MediaBreakpoints";
 
 import Header from "../common/Header";
 import Footer from "../common/Footer";
 import DynamicButton from "../common/DynamicButton";
+import ToggleButton from "../common/ToggleButton";
+import SettingsSectionHeader from "../common/SettingsSectionHeader";
+
+const Wrapper = styled.div`
+  max-width: 1920;
+  margin: auto;
+  margin-top: 100px;
+  margin-left: 15%;
+  margin-right: 15%;
 
-class Settings extends Component {
-  constructor(props) {
-    super(props);
+  @media ${device.mobileM} {
+    margin-top: 50px;
+    margin-left: inherit;
+    margin-right: inherit;
   }
+`;
+const PageTitle = styled.h2`
+  font-size: 35;
+
+  @media ${device.mobileM} {
+    display: none;
+  }
+`;
+
+const Settings = () => {
+  const [emailNotification, setEmailToggle] = useState(false);
+  const [phoneNotification, setPhoneToggle] = useState(false);
 
-  render() {
-    return (
-      <>
-        <Header />
+  return (
+    <>
+      <Header />
+      <Wrapper>
+        <PageTitle>Notification Settings</PageTitle>
         <div
           style={{
-            maxWidth: 1920,
-            margin: "auto",
+            margin: 100,
           }}
         >
```

Comment:
Need to convert to a styled component to add media queries

Will be addressed in #23